tests/utils/cn.test.ts

```
@@ -0,0 +1,23 @@
+import { cn } from '@/utils/cn';
```

ESLint

[error] Expected 2 empty lines after import statement not followed by another import.

(import/newline-after-import)